src/cuda_ts.h

```diff
@@ -3,10 +3,11 @@
 
 struct CUDAThreadState: ThreadState{
 
-    Task *launch(Kernel kernel, uint32_t size,
-                 std::vector<void *> *kernel_params,
-                 uint32_t kernel_param_count,
-                 const uint8_t *kernel_params_global) override;
+    void barrier() override{}
+
+    Task *launch(Kernel kernel, KernelKey *key, XXH128_hash_t hash,
+                 uint32_t size, std::vector<void *> *kernel_params,
+                 const std::vector<uint32_t> *) override;
```

Comment:
Name of the last argument is missing.
You can comment it `/* kernel_params_global */` if you want to indicate it's unused, but typically I haven't seen that done in method declarations.